patterns/template-search-portfolio.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Title: Portfolio Search Template
+ * Slug: twentytwentyfour/template-search-portfolio
+ * Template Types: search
+ * Viewport width: 1400
+ * Inserter: no
+ */
+?>
+
+<!-- wp:group {"style":{"position":{"type":"sticky","top":"0px"}},"layout":{"type":"default"}} -->
+<div class="wp-block-group">
+	<!-- wp:template-part {"slug":"header","theme":"twentytwentyfour"} /-->
+</div>
+<!-- /wp:group -->
+
+
+<!-- wp:group {"tagName":"main","align":"full","layout":{"type":"constrained"}} -->
+<main class="wp-block-group alignfull">
+	<!-- wp:query-title {"type":"search","align":"wide","style":{"typography":{"lineHeight":"1"},"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|10"}}}} /-->
+
+	<!-- wp:columns {"align":"wide"} -->
+	<div class="wp-block-columns alignwide">
+		<!-- wp:column {"width":"66.66%"} -->
+		<div class="wp-block-column" style="flex-basis:66.66%">
+			<!-- wp:search {"label":"Search","showLabel":false,"width":null,"widthUnit":"%","buttonText":"Search","fontSize":"medium"} /-->
```

Comment:
I wonder if we should have a hidden pattern for this for translation sake

Reply:
Yeah that would be nice.

Reply:
yeah this whole column can be moved to a pattern and reused.

Reply:
Ah, the column is not used in search-writer, or 404, so, just the single block then.

Reply:
Just merged your other PR, you can update this one now